javascripts/projectsService.js

```diff
@@ -5,16 +5,17 @@
 /* eslint function-paren-newline: [ "off" ] */
 /* eslint implicit-arrow-linebreak: [ "off" ] */
 /* eslint no-confusing-arrow: [ "off" ] */
+/* eslint no-var: [ "error" ] */
 
 // @ts-nocheck
 
 // required for loading into a NodeJS context
 if (typeof define !== 'function') {
-  var define = require('amdefine')(module);
+  define = require('amdefine')(module);
```

Comment:
This is an exception that we need to figure out how to handle better:

```suggestion
  /* eslint-disable-next-line no-var */
  var define = require('amdefine')(module);
```